hls4ml/model/graph.py

```diff
@@ -712,7 +712,7 @@ class TraceData(ctypes.Structure):
         layer_sizes = {}
         n_traced = 0
         for layer in self.get_layers():
-            if layer.get_attr('function_cpp', None) and layer.get_attr('Trace', False):
+            if layer.get_attr('function_cpp', None) and layer.get_attr('trace', False):
```

Comment:
Does this mean the key in the config is now called `'trace'` instead of `'Trace'`? Or is this just related to the `snakeCase -> pascal_case` internal translation? Basically, does the way a user manipulates the configuration keys stay the same?

Reply:
Yes, 'Trace' becomes `trace` because otherwise there would be both `Trace` and `trace` in the attribute list. This is just an interim solution, in the future I hope we can move to pascal_case everywhere.